main.ts

```diff
@@ -17,9 +19,20 @@ async function handleFolderOpen() {
     }
 }
 
+let win: BrowserWindow | null = null;
+
+let logAndSend = (msg: string) => {
+    log.info(msg);
+    if (win) {
+        console.log("now sending the message to the frontend");
+        win.webContents.send('send-to-frontend-channel', {message: msg, level: 'info'});
+    }
+
+}
+
 
 let createWindow = () => {
```

Comment:
can we use object oritented code here, lets create a class 

PhotoOrganizerServer and then create methods on it like 'init' and 'start' etc.